client/src/calendar.jsx

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable class-methods-use-this */
```

Comment:
Did you actually want to commit this with the eslint disabled?

Reply:
oops...I'll remove. Thanks!